VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistory.tsx

```diff
@@ -404,11 +403,17 @@ function PrescriptionHistory({ navigation, route }: PrescriptionHistoryProps) {
       a11yLabel: a11yLabelVA(t('goToMyVAHealth')),
     }
 
-    const props: CollapsibleAlertProps = {
-      border: 'warning',
-      headerText: t('prescription.history.transferred.title'),
-      body: (
-        <>
+    return (
+      <Box mt={theme.dimensions.standardMarginBetween}>
+        <AlertWithHaptics
+          variant="warning"
+          expandable={true}
+          focusOnError={false}
+          header={t('prescription.history.transferred.title')}
+          headerA11yLabel={t('prescription.history.transferred.title')}
+          analytics={{
+            onExpand: () => logAnalyticsEvent(Events.vama_cerner_alert_exp()),
+          }}>
           <TextView
```

Comment:
Could potentially be partially simplified moving this to the `description` prop to AlertWithHaptics.